google-cloud-bigquery/lib/google/cloud/bigquery/convert.rb

```diff
@@ -96,7 +108,12 @@ def self.format_value value, field
           elsif field.type == "BYTES"
             StringIO.new Base64.decode64 value[:v]
           elsif field.type == "TIMESTAMP"
-            ::Time.at Float(value[:v])
+            if is_int?(value[:v])
```

Comment:
I see golang just threw out their ParseFloat in https://github.com/googleapis/google-cloud-go/pull/9368/files so the implementations seem to be at odds. I'd like to see some comments and a documentation link (preferably also in comments) describing what we expect to see when parsing timestamps.